block_additional_mappings/smpte12m_timecode.md

```diff
@@ -0,0 +1,51 @@
+## Block Additional Mapping - SMTPE 12M Timecode
+
+### Timecode Description
+
+SMPTE 12 timecode values can be stored in the `BlockMore Element` to associate the content of a Matroska Block with a particular timecode value.
+
+The Block Additional Mapping contains a full binary representation of a 64 bit SMPTE timecode value expressed exactly as defined in Section 7 and 8 of SMPTE 12M [@!SMPTE.12M.2014]. For convenience, here are the bit assignments from SMPTE 12M, for information:
+
+| Bit Positions | Label                  |
+|:-------------:|:-----------------------|
+| 0--3          | Units of frames        |
+| 4--7          | First binary group     |
+| 8--9          | Tens of frames         |
+| 10            | Drop frame flag        |
+| 11            | Color frame flag       |
+| 12--15        | Second binary group    |
+| 16--19        | Units of seconds       |
+| 20--23        | Third binary group     |
+| 24--26        | Tens of seconds        |
+| 27            | Polarity correction    |
+| 28--31        | Fourth binary group    |
+| 32--35        | Units of minutes       |
+| 36--39        | Fifth binary group     |
+| 40--42        | Tens of minutes        |
+| 43            | Binary group flag BGF0 |
+| 44--47        | Sixth binary group     |
+| 48--51        | Units of hours         |
+| 52--55        | Seventh binary group   |
+| 56--57        | Tens of hours          |
+| 58            | Binary group flag BGF1 |
+| 59            | Binary group flag BGF2 |
+| 60--63        | Eighth binary group    |
```

Comment:
Given it's "codec" opaque data format for Matroska, you should specify if it's in Little Endian or Big Endian.

Reply:
thx, ping to @JeromeMartinez for a suggestion here.

Reply:
For reference it is big endian in e.g. Dolby E, DV, GXF, MXF (SDTI or Ancillary data), MPEG-4 Visual.
So please big endian.

Reply:
@dericed apologizes I wrote a bit too quickly.
In the listed formats, time code is not exactly SMPTE ST 12-1 (fields order are different, and none have the same order...), by "big endian" I mean that most significant bit of a byte is the most significant bit of tens or units, this is not like in Vorbis (the only format I am aware of that inverts totally the bits).
But "bit 0" is the most significant bit ("Units of frames 8" in SMPTE ST 12-1, which is more oriented to computers), so "big endian" wording may be misleading. maybe saying something like that bit 0 in this list is mapped to the most significant bit of the first byte.

Note that RFC 5484 does not make it explicit.

Reply:
@JeromeMartinez  On page 15 of SMPTE ST-12-1, referencing the storage of the time address it says "The lowest numbered bit of each group corresponds to the least significant bit of each BCD digit" and on page 27, it referencing to the storage of the 4 bits within the binary groups and says "The lowest numbered bit of each group corresponds to the least significant bit of that group." Doesn't that imply that 'big 0' is the least significant bit ("Units of frames 1")?